manage.py

```diff
@@ -26,6 +26,21 @@ def make_shell_context():
 manager.add_command('db', MigrateCommand)
 
 
+def create_default_admin(first_name, last_name, email, password):
+    if (User.query.filter_by(email=email).first() is not None):
+        return
```

Comment:
Why don't you move this logic to the `setup_dev` function below. That way we only `create_default_admin` if that account doesn't exist already.